scripts/ctest/ctest_driver_test.py

```diff
@@ -39,6 +39,8 @@ def main(build_path, ctest_executable):
                     args_to_send.append("--no-gpu")
             result = subprocess.check_output(args_to_send, shell=False, cwd=build_path, stderr=sys.stderr)
             output = result.decode('utf-8')
+            print(f"Executed: {args_to_send}")
+            print(output)
```

Comment:
This seems like it could get a bit chatty. Do we need all the output?

Reply:
oh, good catch, thanks, removing, was debugging...